### PR TITLE
added the API function to the add people to channel button

### PR DESCRIPTION
**Pull request by JideijeEmeka**
Found no API integration for the "ADD" button in the add people to channel UI.
Therefore using the swagger api endpoint i was able to add the API function in the button that will do the magic .

### CODE REVIEW

File: lib/ui/view/channel/channel_add_people_view.dart [hunk 1]
```diff
@@ -39,7 +39,11 @@ class ChannelAddPeopleView extends StatelessWidget {
                     .make(),
                 highlightColor: Colors.transparent,
                 splashColor: Colors.transparent,
-                onTap: () {},
+                onTap: () async {
+                  await addPeople().then((value) {
+                    print(value);
```
Comment (pauleke65) [inline lib/ui/view/channel/channel_add_people_view.dart:44]: Remove all print statements.
Use the logger if needed

File: lib/ui/view/channel/channel_add_people_view.dart [hunk 2]
```diff
@@ -128,4 +132,28 @@ class ChannelAddPeopleView extends StatelessWidget {
...
-}
\ No newline at end of file
+}
+
+Dio dio = new Dio();
+Future addPeople() async {
+  final String pathUrl =
+      'https://channels.zuri.chat/api/v1/614679ee1a5607b13c00bcb7/channels/614679ee1a5607b13c00bcb7/members/';
+
+  dynamic data = {
```
Comment (pauleke65) [inline lib/ui/view/channel/channel_add_people_view.dart:142]: Why is data dynamic and not a Map?

**Review by pauleke65 — CHANGES_REQUESTED** [review]
Changes Requested:
Please move all logic to the view model and make requested changes

**Comment by JideijeEmeka** [comment]
> Changes Requested: Please move all logic to the view model and make requested changes

**Comment by JideijeEmeka** [comment]
Okay

**Pull request closed without merging**